subscription_integration_test.go

```diff
@@ -745,3 +745,104 @@ func TestSubscriptionAllFieldsWithTrialPeriodNeverExpires(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestSubscriptionModifications(t *testing.T) {
```

Comment:
I could flesh out this test so that it exercised Update and Remove, I'll push some additional tests for this.